src/pages/api/stats/calls/date.ts

```diff
@@ -0,0 +1,111 @@
+import { createApiFetch } from 'utils/apiFetch';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+export interface DateStats {
```

Comment:
DateStats is exported from both date.ts and hour.ts , is this intentional or could it instead be created in only one place and imported?

Reply:
Yeah you're right. This should be cleaned up in general, because a large portion of the code is reaused across `date` and `hour`. Do you think that needs to happen before merge?

Reply:
nah, get it in there and we can clean it later! :D